plugins/cricket-scoreboard/manager.py

```
@@ -0,0 +1,470 @@
+            return False
+        teams = cfg.get("favorite_teams", [])
+        if teams and not isinstance(teams, list):
+            return False
+        return True
+
+    def on_config_change(self, new_config: Dict[str, Any]) -> None:
+        if BasePlugin:
+            try:
+                super().on_config_change(new_config)
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)